persistence/src/multi-jvm/scala/com/lightbend/lagom/javadsl/persistence/ClusteredPersistentEntitySpec.scala

```diff
@@ -73,6 +78,13 @@ object ClusteredPersistentEntityConfig extends MultiNodeConfig {
       """)
   }
 
+  nodeConfig(node3) {
+    ConfigFactory.parseString(
+      s"""
+       akka.cluster.roles = ["read-side"]
+      """).withFallback(PersistenceSpec.config("ClusteredPersistentEntitySpec"))
+  }
```

Comment:
Node 3 is now a read side node in addition to node 1, but doesn't hold entities.